packages/design-system-tailwind-preset/src/index.test.ts

```diff
@@ -1,8 +1,75 @@
 import tailwindConfig from '.';
```

Comment:
I think these tests are a little more straightforward to follow and have good explanations in the descriptions of the tests so not adding many comments here